tests/virt/node/hyperv_support/test_hyperv_features_in_vm.py

```diff
@@ -56,17 +56,6 @@ def verify_evmcs_related_attributes(vmi_xml_dict):
     assert vapic_hyperv_feature["@state"] == "on", f"vapic feature in libvirt: {vapic_hyperv_feature}"
 
 
-@pytest.fixture(scope="module")
-def skip_if_no_evmcs_support(schedulable_nodes):
-    for node in schedulable_nodes:
-        if any([
-            label == "cpu-feature.node.kubevirt.io/vmx" and value == "true" for label, value in node.labels.items()
-        ]):
-            return
-
-    pytest.skip("Cannot run the test, none of the nodes has vmx support.")
```

Comment:
same here - I would prefer to keep this verification and fail with clear message when no vmx support

Reply:
+1

Reply:
we may run tests on other clusters: IBM, AWS.. who knows what else will be added in future
if the checking on label is still relevant - I would prefer to keep it, potentially it may save debugging time

Reply:
ack